src/chapter-content/Chapter N: Sample.md

```diff
@@ -0,0 +1,59 @@
+“This is a quote.”
+— Name
+## Section Header
+Paragraph of text
+
+
+Another paragraph of text
+
+ ``` 
+// This is code block
+let pos = createVector(x, y);
+ ``` 
+
+Another paragraph of text
+
+>This is an extra callout box of text not part of the regular flow.
```

Comment:
The HTML version of this would be:

```html
<div data-type="note">
This is an extra callout box of text not part of the regular flow.
</div>
```

I also have:

```html
<span class="highlight">This is highlighted text.</span>   
```